code.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "name": "ds-pipelines-targets-example-wqp",
+    "organization": "U.S. Geological Survey",
+    "description": "An example targets pipeline for pulling data from the Water Quality Portal (WQP)",
+    "version": "main",
+    "status": "Development",
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://github.com/USGS-R/ds-pipelines-targets-example-wqp/blob/main/LICENSE.md"
+        }
+      ]
+    },
+    "homepageURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/",
+    "downloadURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/archive/refs/heads/main.zip",
+    "disclaimerURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/blob/main/DISCLAIMER.md",
+    "repositoryURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp.git",
+    "vcs": "git",
+    "laborHours": 100,
+    "tags": [
+      "R",
+      "USGS"
+    ],
+    "languages": [
+      "R"
+    ],
+    "contact": {
+      "name": "Lauren Koenig-Snyder",
+      "email": "lkoenigsnyder@usgs.gov"
+    },
+    "date": {
+      "metadataLastUpdated": "2023-03-07"
+    }
+  },
+  {
+    "name": "",
+    "organization": "U.S. Geological Survey",
+    "description": "",
+    "version": "v0.1.0",
+    "status": "Ideation",
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "LICENSE.md"
+        }
+      ]
+    },
+    "homepageURL": "",
+    "downloadURL": "",
+    "disclaimerURL": "",
+    "repositoryURL": "",
+    "vcs": "git",
+    "laborHours": 300,
+    "tags": [
+      "R",
+      "USGS"
+    ],
+    "languages": [
+      "R"
+    ],
+    "contact": {
+      "name": "",
+      "email": ""
+    },
+    "date": {
+      "metadataLastUpdated": "2022-10-16"
+    }
+  }
```

Comment:
Just curious, is there a reason why a lot of these fields are duplicated from above (and contain different values)?

Reply:
Based on what I've seen from a few other DOI-USGS repos, repos with multiple versions include this. Laura's [code.json](https://github.com/DOI-USGS/dataRetrieval/blob/main/code.json) includes info for the development branch and the release version. I wasn't quite sure what to do here.

Reply:
Ah, OK. [Here](https://github.com/DOI-USGS/nhdplusTools/blob/main/code.json)'s an example from nhdplusTools where Dave stacks the metadata for successive versions.